source/contracts/Controlled.sol

```diff
@@ -36,7 +37,16 @@ contract Controlled is IControlled {
         return true;
     }
 
-    function suicideFunds(address _target) public onlyControllerCaller returns(bool) {
+    function suicideFunds(address _target, ERC20Basic[] _tokens) public onlyControllerCaller returns(bool) {
+        // Transfer tokens to target
+        // TODO Uncomment when Market.sol size is fixed
```

Comment:
Add a `:` after `TODO` for `// TODO: Uncomment...`.  This will make it so it shows up when we are searching for TODOs as that is how the rest are formatted.

Reply:
(assuming your plan is to merge with the TODO included)